app/models/story_distributions/episode_distribution.rb

```diff
@@ -10,13 +10,26 @@ def distribute!
     add_episode_to_feeder
   end
 
-  def add_episode_to_feeder
+  def add_episode_to_feeder(create_attributes = {})
     return unless url.blank?
     podcast = distribution.get_podcast
-    episode = podcast.episodes.post(episode_attributes)
+    episode = podcast.episodes.post(episode_attributes.merge(create_attributes))
     episode_url = URI.join(feeder_root, episode.links['self'].href).to_s
     raise 'Failed to get episode url on create' if episode_url.blank?
     update_attributes!(url: episode_url) if episode_url
+    episode
+  end
+
+  def get_episode
```

Comment:
I was using this until I refactored to do a single create rather than a create then update for the feeder episode.

I could delete it, but it is convenient, and mirrors `get_podcast` from the `PodcastDistribution`